.github/actions/build-gstreamer-for-windows/action.yml

```
@@ -199,6 +209,10 @@ runs:
         echo "vs_install_version = \"vs17\"" >> localconf.cbc
         echo "num_of_cpus = ${NUMBER_OF_PROCESSORS}" >> localconf.cbc
         echo "package_origin = \"${CERBERO_PACKAGE_ORIGIN}\"" >> localconf.cbc
+
+        # Switch gst-plugins-rs provider to blinemedical
+        echo "recipes_remotes = {'gst-plugins-rs': {'patched': '${GST_PLUGINS_RS_SOURCE}'}}" >> localconf.cbc
+        echo "recipes_commits = {'gst-plugins-rs': '${GST_PLUGINS_RS_REF}'}" >> localconf.cbc
```

To support allowing these to be blank:
```suggestion
        # Handle custom gst-plugins-rs provider
        if [ -n "${GST_PLUGINS_RS_SOURCE}" ] && [ -n "${GST_PLUGINS_RS_REF}" ]; then
          echo "recipes_remotes = {'gst-plugins-rs': {'patched': '${GST_PLUGINS_RS_SOURCE}'}}" >> localconf.cbc
          echo "recipes_commits = {'gst-plugins-rs': '${GST_PLUGINS_RS_REF}'}" >> localconf.cbc
        fi
```